admin/plugins.php

```diff
@@ -55,6 +55,20 @@ function perflab_get_standalone_plugins() {
 	);
 }
 
+/**
+ * Returns an array mapping of standalone plugins main file to existing wpp modules.
+ *
+ * @since n.e.x.t
+ *
+ * @return string[]
+ */
+function perflab_get_standalone_plugins_file_map() {
+	return array(
+		'webp-uploads'          => 'webp-uploads/load.php',
+		'dominant-color-images' => 'dominant-color-images/load.php',
+	);
+}
```

Comment:
I don't think this function is needed. We could probably just use `{slug}/load.php`?

Alternatively we could add support to activate just via the slug to the AJAX PHP functions. That way we wouldn't need to guess the plugin main file name at all, and it could be detected properly via the PHP function after the plugin was already installed.

The latter is maybe a bit too much effort for the benefit, so maybe we just go with the assumption of `{slug}/load.php`?

Reply:
When initially working on another PR, I considered using `{slug}/load.php`. However, upon examining the main file for the 'Performant Translations' plugin, I found it as `performant-translations/performant-translations.php`. That's why I included the mapping function. Happy to update if it safe?

Reply:
Performant Translations is not an existing module, so is that even relevant here?

Reply:
Nope. But in future if someone add new module with different file name then it create bug. `Performant Translations` is just example.

Reply:
> Nope. But in future if someone add new module with different file name then it create bug.

Do you mean a new module or a new standalone plugin? For modules `load.php` is the standard, so we wouldn't use a different file name there. And for standalone plugins this function here seems not relevant? Or what am I missing?

Reply:
@mukeshpanchal27 @swissspidy Actually I think for this functionality Performant Translations _does_ matter. This PR is about installing and activating a standalone plugin, regardless of whether or not it is for an existing module. Performant Translations is a plugin that we want to link to here, so that needs to be covered. This is different in #899, which is only about plugins that _have_ an equivalent module.

That said, I still don't think we should need this function. How about the following instead:
* We enhance the logic for the `perflab_activate_plugin` action.
* Normally it is passed the plugin basename (e.g. `webp-uploads/load.php`). We could add support for alternatively receiving just the plugin slug (e.g. `webp-uploads`).
* Since at that point the plugin would already have been installed, we can simply look up the full basename. For example, call `get_plugins()` and find the one that is `webp-uploads/{mainfile.php}`. Then we have the basename dynamically, without a need to hardcode it.
* It also simplifies the behavior on the client, basically we don't need to know the basename on the client in JS, as the `perflab_activate_plugin` action on the server just figures it out automatically.

Reply:
@felixarntz @swissspidy i have added plugin activation support using the module slug as suggested.